### ISSUE TITLE
RFE: consider bumping virtualbox bits in kernel-lts to 6.0

### ISSUE THREAD

**Issue by AntonioMeireles** [clearlinux/distribution#342]
Hi all!, 

Current `kernel-lts` is shipping an outdated version of the VirtualBox kmods (5.2.16 r123759 when 6.0 is already GM), which is clearly sub-optimal, specially if one is already using latest VirtualBox, so bumping those, which should herald no side effects and be backwards compatible (with older VB versions) would be great.

Thanks in Advance!

António

**Comment by miguelinux** [comment]
We have updated to 6.0.4.

Could you test with the current version?

Thanks

**Comment by AntonioMeireles** [comment]
hi!

humm - this one is interesting. it rants (see bellow) but the shipping kmods are actually / already from 6.0.4. looking / digging. will report in a bit. 

```
make VERSION=27720 test-virtualbox                                                                                                                                                                                                                     ==> box: Box file was not detected as metadata. Adding it directly...
==> box: Adding box 'clear-test' (v0) for provider: virtualbox
    box: Unpacking necessary files from: file:///Users/am/hacks/ClearLinux-packer/boxes/virtualbox/ClearLinux-27720.virtualbox.box
==> box: Successfully added box 'clear-test' (v0) for 'virtualbox'!
~/hacks/ClearLinux-packer/extras/test ~/hacks/ClearLinux-packer
Bringing machine 'test-box' up with 'virtualbox' provider...
==> test-box: Importing base box 'clear-test'...
==> test-box: Matching MAC address for NAT networking...
==> test-box: Setting the name of the VM: test_test-box_1549895807815_28851
==> test-box: Clearing any previously set network interfaces...
==> test-box: Preparing network interfaces based on configuration...
    test-box: Adapter 1: nat
==> test-box: Forwarding ports...
    test-box: 22 (guest) => 2222 (host) (adapter 1)
==> test-box: Running 'pre-boot' VM customizations...
==> test-box: Booting VM...
==> test-box: Waiting for machine to boot. This may take a few minutes...
    test-box: SSH address: 127.0.0.1:2222
    test-box: SSH username: clear
    test-box: SSH auth method: private key
==> test-box: Machine booted and ready!
==> test-box: Checking for guest additions in VM...
    test-box: The guest additions on this VM do not match the installed version of
    test-box: VirtualBox! In most cases this is fine, but in rare cases it can
    test-box: prevent things such as shared folders from working properly. If you see
    test-box: shared folder errors, please make sure the guest additions within the
    test-box: virtual machine match the version of VirtualBox you have installed on
    test-box: your host and reload your VM.
    test-box: 
    test-box: Guest Additions Version: 5.2.0 r68940
    test-box: VirtualBox Version: 6.0
(...) 
```

**Comment by AntonioMeireles** [comment]
dunno is this is the root case *but* here's something fishy ...
``` 
$ fgrep -ia config_vbox /usr/lib/kernel/config-4.19.20-420.lts 
CONFIG_VBOXGUEST=y
CONFIG_VBOXSF=m
CONFIG_VBOXVIDEO=m
```
```
 find  /usr/lib/modules/4.19.20-420.lts/kernel/drivers/ | grep -ia vbox
/usr/lib/modules/4.19.20-420.lts/kernel/drivers/misc/vboxguest
/usr/lib/modules/4.19.20-420.lts/kernel/drivers/misc/vboxguest/vboxguest.ko
/usr/lib/modules/4.19.20-420.lts/kernel/drivers/misc/vboxsf
/usr/lib/modules/4.19.20-420.lts/kernel/drivers/misc/vboxsf/vboxsf.ko
/usr/lib/modules/4.19.20-420.lts/kernel/drivers/staging/vboxvideo
``` 
i suspect that the fact that vboxguest having dual personality ... (static and as regular kmod) may not be helping here ... or it may be something else ... digging more

**Comment by AntonioMeireles** [comment]
OTOH on dmesg i still get `vboxguest: host-version: 6.0.4r128413` which is *right* ...

**Comment by miguelinux** [comment]
I just removed the in-tree vbox video driver. It will be updated by tomorrow built.

**Comment by iphutch** [comment]
Thanks miguelinux

**Comment by miguelinux** [comment]
fixed at linux-lts-4.19.26-432 and linux-lts2018-4.19.26-3
